package.json

```diff
@@ -621,7 +621,8 @@
                         "type": "array",
                         "default": [
                             "submit",
-                            "test"
+                            "test",
+                            "star"
```

Comment:
No need to add this shortcut by default. But please add it in the items, and also documents in readme